tools/serverpod_cli/lib/src/analyzer/dart/definitions.dart

```diff
@@ -58,6 +58,12 @@ abstract base class MethodDefinition {
   /// The named parameters of this method.
   final List<ParameterDefinition> parametersNamed;
 
+  List<ParameterDefinition> get allParameters => [
```

Comment:
Comment: Nice 🌟

Reply:
Suggestion: Would be nice to use this where possible in the library generator as well.

To ensure this refactor doesn't diverge the code.

Reply:
Good suggestion, will fix!

Reply:
b15a517